patches/chrome-android-java-src-org-chromium-chrome-browser-ChromeTabbedActivity.java.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
+index 41aa93fd0189b45656d7310adcec1c0b7a716d53..85e06e946eb3cfe662448e208d0f1fc1255bb197 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
+@@ -2249,6 +2249,7 @@ public class ChromeTabbedActivity extends ChromeActivity<ChromeActivityComponent
+     }
+ 
+     private void hideOverview() {
```

Comment:
since we've already made this public, why don't we just subclass it and return the subclass in BraveActivity.getChromeTabbedActivity ?

Reply:
Created a follow up issue for that https://github.com/brave/brave-browser/issues/16242